charts/prometheus-logstash-exporter/ci/default-values.yaml

```diff
@@ -0,0 +1 @@
+# Leave this file empty to ensure that CI runs builds against the default configuration in values.yaml.
```

Comment:
imho the file is not needed if empty.
at least i thought the default values are always checked?

Reply:
right, I copy it from elasticsearch chart, will remove it